metadata_service/proxy/gremlin_proxy.py

```diff
@@ -1449,7 +1451,7 @@ def get_latest_updated_ts(self) -> int:
         """
 
         results = _V(g=self.g, label=VertexTypes.Updatedtimestamp,
-                     key=AMUNDSEN_TIMESTAMP_KEY).values('latest_timestamp').toList()
+                     key=AMUNDSEN_TIMESTAMP_KEY).values('latest_timestmap').toList()
```

Comment:
From: https://github.com/amundsen-io/amundsendatabuilder/blob/master/databuilder/models/es_last_updated.py#L19

Reply:
agree, let's fix in databuilder for the typo.

Reply:
Haha fixed!